xcode/Subconscious/Shared/Library/SQLite3Database.swift

```diff
@@ -163,7 +171,7 @@ final class SQLite3Database {
             text(escapePrefixQueryLike(query))
         }
 
-        public func unwrap() -> String? {
+        public func toString() -> String? {
```

Comment:
Introduce named `toX` methods. The older `unwrap` methods relied on Swift to disambiguate type information from return type, which it often failed to do, requiring type annotations. This new method naming convention allows for more fluent chaining without type annotations.